File: tests/fixtures/compare/sample.swift
// Realistic Swift code with various issues
// TODO: refactor

import Foundation

func process(_ items: [Int]) -> [Int] {
    var result: [Int] = []
    for item in items {
        if item > 10 {
            if item > 20 {
                result.append(item)
            }
        }
    }
    return result
}

func empty() {}

func complex(_ a: Int, _ b: Int, _ c: Int, _ d: Int, _ e: Int) -> Int {
    return a + b + c + d + e
}

File: tests/fixtures/compare/sample.scala
// Realistic Scala code with various issues
// TODO: refactor

object Sample {
  def process(items: List[Int]): List[Int] = {
    items.filter(_ > 10)
  }

  def empty(): Unit = {}

  def complex(a: Int, b: Int, c: Int, d: Int, e: Int): Int = {
    a + b + c + d + e
  }
}
